Source/items.cpp

```diff
@@ -2169,10 +2169,11 @@ void SpawnOnePremium(Item &premiumItem, int plvl, int playerId)
 
 	plvl = clamp(plvl, 1, 30);
 
-	int count = 0;
+	int maxCount = 65000;
+	if (gbIsHellfire)
+		maxCount = 150;

```

Comment:
I also think this will lead to items worth more than the cap being offered by Griswold where the game would otherwise have locked up, to avoid reopening #1478 need to accept this possibility and maybe call it out in the change notes?